src/webjs.coffee

```diff
@@ -120,7 +120,7 @@ exports.GenericApp = class GenericApp
     log_request: (req, res, data) ->
         td = (new Date()) - req.start_date
         @log('info', req.method + ' ' + req.url + ' ' + td + 'ms ' +
-                (if res.finished then res._header.split('\r')[0].split(' ')[1] \
+                (if res.finished and res._header then res._header.split('\r')[0].split(' ')[1] \
```

Comment:
This will log `(unfinished)` if `res._header` does not exist, even if `res.finished` is true. Can you fix that?